src/apps/service/apps/de/jobs/common.clj

```diff
@@ -114,6 +114,17 @@
        (take-while (comp nil? :external_app_id))
        (reduce #(.buildStep request-builder %1 %2) [])))
 
+(defn load-htcondor-extra-requirements
```

Comment:
I'd make this function private (`defn-`), just to make it obvious it's not in use outside this namespace.

Reply:
Ah, yeah, good idea.